pkg/cmd/build/build_test.go

```diff
@@ -486,38 +484,38 @@ func TestExtractFromContextAndDockerfile(t *testing.T) {
 		expectedError      string
 		dockerfilesCreated []string
 	}{
-		{
```

Comment:
@jLopezbarb what are we going to do with the commented test?

Reply:
I forgot to remove the comment. Tanks for sporting it